### DISCUSSION TITLE
Detailslist: ColCellType:"clickableimage" with border of ColCellType:"indicatortag"

### DISCUSSION THREAD

**Discussion by velsreis** [microsoft/powercat-creator-kit#296] [Q&A]
![image](https://user-images.githubusercontent.com/78064698/223131883-c940a9b0-2d09-4592-a254-89397696df39.png)

![image](https://user-images.githubusercontent.com/78064698/223132025-2cad8112-aaf5-448a-ad98-73d318898c05.png)

This Monday morning when I opened Power Apps to work on my application, all the clickable image elements of all the Detailslist components from my app had the borders of the indicatortag. I need to remove them. Can anyone help me?

Last Friday it was working normally, all the elements without borders as it should be.

ps I restored the last version from Friday and it went back to normal, but I would like to understand what happened.

**Comment by yaseralhosani** [comment]
"tag" "indicatortag" is default with a gray border and we can't change it yet.

"clickableimage" is without a border, please recheck your code / refresh as well as re-add the fields. 

Below example code
```
{
        ColName: "iconimage",
        ColDisplayName: "Acoes",
        ColCellType: "clickableimage",
        ColImageWidth: 16,
        ColWidth: 48
}
```

**Reply by velsreis** [reply]
I’m using clickableimage anyway, it usually comes without the border as it should be but sometimes it bugs and gets this gray border 
then I need to go back a version to get back to normal

the normal and expected thing is for him to appear like this
![image](https://user-images.githubusercontent.com/78064698/224104119-75087d07-1b87-400a-b351-785bc18514cc.png)
